python/tvm/tir/schedule/schedule.py

```diff
@@ -2852,7 +2852,7 @@ def transform_layout(
             The buffer to be transformed, or a specification of how to
             identify the buffer to be transformed.
 
-            If `buffer` if a tuple of ``(str,int)``, the first item
+            If `buffer` is a tuple of ``(str,int)``, the first item
```

Comment:
This is an unrelated change, please put it in a separate PR.

Reply:
Removed.